.github/workflows/ci.yml

```
@@ -15,13 +15,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up JDK 22
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v4
```

actionlint

missing input "distribution" which is required by action "actions/setup-java@v4". all required inputs are "distribution"

(action)